.github/workflows/linux.yml

```diff
@@ -297,6 +302,9 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+      - name: Get TinyGo version
+        id: version
+        run: ./.github/workflows/tinygo-extract-version.sh >> "$GITHUB_OUTPUT"
```

Comment:
Above you used `tee -a` instead of `>>`.  Why the change?

Reply:
Fixed. This was the older code. I updated the code in most places (from `>>` to `tee -a` to be able to see the output of the step), but apparently missed this one.